MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: Manually create view items in .NET MAUI DataForm control | Syncfusion
```

Comment:
title: Manually create data editors in .NET MAUI DataForm control | Syncfusion
description: Learn about the working with manually or explicitly create data editor in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications.